resources/mac-address.json

```diff
@@ -0,0 +1,3 @@
+{
+  "gcs": "0013A20040917A31"
```

Comment:
please make this a list of possible Mac addresses. 

The code should be able to go through all the possible Mac addresses that the GCS can be and pick the first that works.

Since vehicles will likely need this information too, we should decide what the radios we will use are and make it run with any of the possible radios.

Reply:
I do not think we need a list of possible MAC addresses. Each platform will have a specific Xbee that will be mounted on the vehicle, as well as extra Xbees for devleopment (in the case of GCS, we will have one of the 6 new Xbees be the field test Xbee).

Now to have everyone know what platforms have what Xbees, here's what I propose:
1. Put a table of everyone's MAC addresses on the Wiki
2. Have a file called `mac-addresses.txt` in the root of [NGCP/Communication](https://github.com/NGCP/Communication). I remember other languages struggle to process JSON files; but I mean if they have a JSON to class converter it shouldn't be too bad right?

Reply:
Remind me to add your rule on class organization to the wiki somewhere.

> Sorry I found a couple more things.
> 
> Specifically for the class organization, they should be specified in the following order:
> 
> 1. all static field variables
> 2. remaining (non static) variables
> 3. all the field variable lambdas (`myFunc = () => { ... }`)
> 4. the contractor or similar to constructor functions
> 5. all the normal methods

Reply:
Also I remember why I put some variables after certain functions. It's cause of the linter... You think I should just disable the rule? See image below:

<img width="663" alt="screen shot 2019-02-08 at 4 44 12 pm" src="https://user-images.githubusercontent.com/13097797/52513621-dd6af400-2bc0-11e9-9021-0218b492e789.png">

Reply:
I don't like having method and variables mixed. I'm all for disabling that rule, unless there's like a solid reason why it needs to be in that order.

So why I want a list of  possible MAC addresses: the xbee im gonna use for GCS is not the XBee your gonna use. If we don't want to manipulate config files (which I agree with you there, we don't want to) then we need to either add ui element to allow the user to input the Mac address, or we detect the xbee radio's Mac address, or we establish a list of the possible xbee gcs radios (the ones you will use for gcs and the ones I will use for gcs) and then add them all to the config file and have it figure out the one it needs.

Reply:
Sure. We can do it like that for development purpose. I assume for all other platforms we will stick with one value

Reply:
For the rule, since it enacts itself on React components only, I will modify the rule to make it adapt to your rules. I wonder if there's a rule that will enforce structuring a class the way you do it. Otherwise we will have developers that will be creating them wrong.

Reply:
Updated rules and mac-addresses

Reply:
I realized that we don't actually need a list of MAC addresses for GCS, since we aren't sending messages to ourselves. It is other platforms that will be sending messages to us, so that will be them that needs those MAC addresses.

When that case happens, either they can implement having a list of GCS MAC addresses, or just have them put the Xbee MAC address we are using for field test (I prefer the latter).

I'll be removing the gcs field in the json file in a future commit, as it is not needed.